Documents/Examples/Examples.md

```diff
@@ -6,7 +6,7 @@
 
 ## `KryptonForm` Fade Effects
 
-![](https://github.com/Krypton-Suite/Extended-Toolkit/blob/version-next-effects/Assets/Examples/KryptonFade.gif)
+![](https://github.com/Krypton-Suite/Extended-Toolkit/blob/master/Assets/Examples/KryptonFade.gif)
```

Comment:
Cannot be `master` - Has to be `Version-next`
Unitl everything is then merged into master !

Reply:
I can move the gif to the `version-next` branch, but as soon it is merged with `master` it'll break.